Neos.Flow/Classes/Session/Aspect/LoggingAspect.php

```diff
@@ -25,21 +25,11 @@
 class LoggingAspect
 {
     /**
+     * @Flow\Inject(name="Neos.Flow:SecurityLogger")
```

Comment:
This changes the Logger from System to Security, on purpose? Thats pretty breaking.

Reply:
👀 very good catch! That's why reviews are important!
```suggestion
     * @Flow\Inject(name="Neos.Flow:SystemLogger")
```